tests/integration/README.md

```diff
@@ -31,8 +32,9 @@ $ make integration TESTPATH="/checkpoint.bats"
 ```
 
 
-To run them on your host, you will need to setup a development environment plus
-[bats](https://github.com/sstephenson/bats#installing-bats-from-source)
+To run them on your host, you need to set up a development environment plus
+[bats (Bash Automated Testing System)](https://github.com/bats-core/bats-core#installing-bats-from-source).
+
```

Comment:
There is an example a few lines below (which is out of this patch context so I can't comment on it) which needs to be updated as well.

Reply:
arf; thought I updated that one; good catch!